app/scripts/models/account.js

```diff
@@ -110,7 +113,7 @@ define([
     isEmpty: function () {
       var self = this;
       return ! _.find(ALLOWED_KEYS, function (key) {
-        return typeof self.get(key) !== 'undefined';
+        return self.get(key) !== DEFAULTS[key];
```

Comment:
This update alters the semantics of the function in a way that no longer matches the name. If a default value is truthy, it would be confusing if `isEmpty` returns true. Perhaps alter the function name or add a new function?